src/drt/src/pa/FlexPA.cpp

```diff
@@ -71,6 +71,13 @@ void FlexPA::setDebug(frDebugSettings* settings, odb::dbDatabase* db)
 void FlexPA::init()
 {
   ProfileTask profile("PA:init");
+  for (auto& master : design_->getMasters())
+    for (auto& term : master->getTerms())
+      for (auto& pin : term->getPins())
+        pin->clearPinAccess();
+  for (auto& term : design_->getTopBlock()->getTerms())
+    for (auto& pin : term->getPins())
+      pin->clearPinAccess();
```

Comment:
Why clear all pin access and not just the the instances being computed?

Reply:
When I thought of it, It didn't make sense that I would remove access points of instances of the target masters and not instances of other masters. For example, let's just say I have:
- master m1: instances i1, i2
- master m2: instances i3, i4
After general pin access, If I call pin access on i1 only, I will have to clear pin access of m1. in the end I will have pin access of i1, i3, and i4. But why not i2? The user won't understand that it's because i2 has the same master of i1 that was cleared at the beginning of pin access, right?